packages/firebase_dynamic_links/README.md

```diff
@@ -97,7 +97,8 @@ Receiving dynamic links on *iOS* requires a couple more steps than *Android*. If
 applinks:YOUR_SUBDOMAIN.page.link
 ```
 
-4. To receive a dynamic link, call the `retrieveDynamicLink()` method from `FirebaseDynamicLinks`:
+4. To receive a dynamic link, call the `getLaunchLink()` method from `FirebaseDynamicLinks` and
```

Comment:
I would include that get launch link only retrieves the link that opens the app.

Reply:
Done!